images/router/haproxy/reload-haproxy

```diff
@@ -12,8 +12,13 @@ if [ -f $pid_file ]; then
   old_pid=$(<$pid_file)
 fi
 
+# Assumes that the docker ip is the first entry in the hosts file.
+# This port is non-configurable via the router cmd, it is HAProxy specific.  It should
+# match the port used in the haproxy-config.template
+peer_name=$(awk 'NR==1 {print $1}' /etc/hosts):1937
```

Comment:
@smarterclayton - been trying to sort this out this morning.

1.  The downward api does not support status.podIP
1.  Adding the support for status.podIP locally resulted in an empty env var

It looks like the pod is still in pending state with no ip when the downward api vars values are injected.  So I'm looking for a short term, acceptable solution.  I could inject the client into the plugin and query based on the downward api's name/namespace.
```
E0519 14:37:49.941356   23572 kubelet.go:899] ------------------- makeing field path for status.podIP with pod api.Pod .
...snip...
 Status:api.PodStatus{Phase:"Pending", Conditions:[]api.PodCondition(nil), Message:"", HostIP:"", PodIP:"", ContainerStatuses:[]api.ContainerStatus(nil)}}
```